andela/mrm_api
# #163448618 Implement clean up of deleted records from the db periodically

#### What does this PR do?
- Implement the permanent deletion of data that had been flagged for deletion in the database.

#### Description of Task to be completed?
- Implemented the deletion of data that had been flagged for deletion by providing a file that can be run whenever the admin requires a database clean up.

#### How should this be manually tested?
- Pull this branch
- Ensure you have data in the database that has been flagged for deletion (by having state set to deleted) 
- Run the command `python run_mrm_services.py cleandb`
- Check to confirm that the data that had been flagged for deletion has been permanently deleted from the database.

#### Any background context you want to provide?
- When permanently deleting, if the data to be deleted has any children they will also be deleted in a cascading manner.

#### What are the relevant pivotal tracker stories?
[#163448618 ](https://www.pivotaltracker.com/story/show/163448618)
#### Screenshots (if appropriate)
- N/A
#### Questions:
